wagtail/documents/templates/wagtaildocs/multiple/add.html

```diff
@@ -10,7 +10,13 @@
 {% endblock %}
 
 {% block main_content %}
-    <div class="drop-zone w-mt-8">
+    <div
+        class="drop-zone w-mt-8"
+        data-controller="w-zone"
+        data-action="dragover@document->w-zone#noop:prevent drop@document->w-zone#noop:prevent dragover->w-zone#activate drop->w-zone#deactivate dragleave->w-zone#deactivate dragend->w-zone#deactivate"
+        data-w-zone-active-class="hovered"
+        data-w-zone-delay-value="10"
```

Comment:
I ended up wrapping these all to new lines like your original suggestion, thanks for proposing that @ayaan-qadri